tests/config/base_ospdo/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
```

Comment:
This is one of the things that may change with #592, we're debating getting rid of kustomize + jinja combination and just going with jinja. If we go with that, you could still have your custom OSPDO j2 file, it would just combine the tls and non-tls parts in one file like #592 does, so hopefully that's not too difficult adjustment.

Reply:
agree, no problem, I can update after the changes as needed